skyuxconfig.json

```diff
@@ -13,7 +13,7 @@
       "browserSet": "speedy"
     },
     "unit": {
-      "browserSet": "paranoid"
+      "browserSet": "speedy"
```

Comment:
@Blackbaud-SteveBrush I have a PR that fixes/disables these tests. Should we undo this change in that PR after this merges?

Reply:
Ship it